src/libs/PolicyUtils.ts

```diff
@@ -303,10 +305,11 @@ function getTagList(policyTagList: OnyxEntry<PolicyTagLists>, tagIndex: number):
     const tagLists = getTagLists(policyTagList);
 
     return (
-        tagLists[tagIndex] ?? {
+        tagLists.at(tagIndex) ?? {
```

Comment:
Please make sure `tagIndex` won't be negative since it can affect the logic then.